app/assets/javascripts/miq_qe.js

```diff
@@ -0,0 +1,25 @@
+if (typeof _ !== 'undefined' && typeof _.debounce !== 'undefined') {
+  var orig_debounce = _.debounce;
+  _.debounce = function(func, wait, options) {
+    var self = this;
+
+    // Override the original fn; new_func will be the original fn with wait prepended to it
+    // We make sure that once this fn is actually run, it decreases the counter
+    var new_func = function() {
+      try {
+        return func.apply(self, arguments);
```

Comment:
I'm thinking self should actually be window here.. if the function passed in as an arg was already bound, then this won't affect it, and if it was not bound, and is accessing this, then what it's expecting to access is window. Or, just pass an empty object because we don't want functions to touch window via this :)